tfjs-converter/src/operations/executors/basic_math_executor.ts

```diff
@@ -159,10 +159,6 @@ export const executeOp: InternalOpExecutor =
               getParamValue('x', node, tensorMap, context) as Tensor)];
         case 'Rsqrt':
           return [ops.rsqrt(getTensor(node.inputNames[0], tensorMap, context))];
-        case 'Prod':
```

Comment:
For removing existing executor: are you sure that there are no old models converted with category alias?

Reply:
I checked an example model file, and we don't actually save the category of an op in the model.json. I don't think this should affect our model support, but good idea to check.

Reply:
yeah `model.json` does not store the category name, my bad. Agreed that it shouldn't affected converted models.